### ISSUE TITLE
Auto-Hide Quest Panel in combat

### ISSUE THREAD

**Issue by denisciumbargi** [GoldpawsStuff/AzeriteUI#127]
Hello again :)

Just a quick note as per some of our previous comments would like, if possible, the option to auto-hide the quest panel while in combat would really make proper room on the screen to focus on the target, especially in raids for example where u need to see more of the area, or even just because I like an almost empty screen where is just yourself and cheeky enemy and you can think better of the many ways to kills them without the clutter around :))

You were saying might cause some issue with bag items for special quests but, and this will show how I have no clue on addons development, there are times buttons are popping up for special spells, could not that button associated with a quest special items pop up next to the map like the special spells too?

Auto-hide being let's say an option in the addon setting rather than always a must could help for those situations, as a turn on turn off thing.

I know might cause you some extra work but as i previously said, we need to challenge ourselves from time to time to keep the brain going ha ha ha :)

Hope this could be done, wishing you a great day!

**Comment by goldpaw** [comment]
I'm _for_ this option. And I'll add in the tooltip that it would require people to manually open the bags and such. Like classic, really. Classic has no quest items in the tracker either.

**Comment by denisciumbargi** [comment]
Classic is always good. 

Actually, something else came up in my mind in relation to this but might only complicate things so feel free to delete and ignore this message if so ha ha, there are some addons, like buttons auto hide that leaves only a little bar or the side and on mouse over opens the panel, would solve the issue of people having to open their bags, always open on mouseover and they can reach it when they need and then will hide for any other occasion. A bit like the player fading option I guess that shows up at mouseover, that would be the easiest probably.

Uploading Rec 0602.mp4…


Tried to do a little screen recording that might better explain my nonsense, check the area between the quest log an map, is where the action happens in the video ha ha

**Comment by goldpaw** [comment]
Yeah, and the pullouts from various addons already exist. I really don't need to remake it in my image. 

I do have a bag addon project with similar functionality in the works, but if that happens or not, I can't say. And definitely not when. I've deleted three versions already, so I'm picky. 

And it's not nonsense, btw. It's great functionality. Don't downtalk yourself in fear of offending. Your ideas are good! ;)
